src/containers/shared/test/utils.test.js

```diff
@@ -205,3 +208,35 @@ describe('utils', () => {
     expect(durationToHuman(300000000)).toBe('9.51 yr.')
   })
 })
+
+describe('AMM utils format asset', () => {
+  it('formats XRP asset', () => {
+    const asset = '10000000000'
+    const formatted = formatAsset(asset)
+
+    expect(formatted).toEqual({ currency: 'XRP' })
+  })
+
+  it('formats non XRP asset', () => {
+    const asset = { currency: 'USD', amount: '100000', issuer: 'your mom' }
+    const formatted = formatAsset(asset)
+
+    expect(formatted).toEqual({ currency: 'USD', issuer: 'your mom' })
+  })
+})
+
+describe('AMM utils localize balance', () => {
+  it('formats XRP balance', () => {
+    const balance = { currency: 'XRP', amount: 9000000 }
+    const formatted = localizeBalance(balance, 'en-US')
+
+    expect(formatted).toEqual('9,000,000')
```

Comment:
The box should be replaced with the actual currency code unicode (`\uE900`)

Reply:
Lets still do this one.  We recently went through and corrected everywhere so dont want to take steps back.

Reply:
done